### PR TITLE
Renaming ItemAdapter class

### LINKED ISSUE #43: Renaming ItemAdapter class
Current ItemAdapter is the adapter class for displaying books, but it isn't named Properly.It should be renamed using Refractor.The name to be given is "BookRecyclerViewAdapter"

### PR DESCRIPTION
**Pull request by misaka3**
## Description
Fix #43 

## Checklist:

- [ ] I used the Reformat code option in android studio for the files I changed
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] The App is running Fine and as Expected In my Emulator or Physical Device
- [x] I have checked my code and corrected any misspellings
- [ ] My changes didn't cause any issues for the other functionalities present before.
- [ ] Requested **reviews** from atleast two **Reviewers**
- [x] This PR is created From a branch which is upto date with the master branch

### CODE REVIEW

**Comment by Amar-2003** [comment]
Hi @misaka3 , don't worry about the checks, the checks contain few errors and is just being setted up

**Comment by Amar-2003** [comment]
You should change only the files ItemAdapter.kt and Book_fragment.kt, revert the other 3 files

File: app/src/main/java/com/example/insights/ForumnActivity.kt [hunk 1]
```diff
@@ -25,11 +25,11 @@ class ForumnActivity : AppCompatActivity() {
...
 
-        val Itemadapter = forumadapter(this, messagedata)
-        recyclerview.adapter = Itemadapter
-        recyclerview.scrollToPosition((Itemadapter.itemCount) - 1)
+        val BookRecylerViewAdapter = forumadapter(this, messagedata)
+        recyclerview.adapter = BookRecylerViewAdapter
+        recyclerview.scrollToPosition((BookRecylerViewAdapter.itemCount) - 1)
         findViewById<EditText>(R.id.message_input).setOnClickListener {
-            recyclerview.scrollToPosition((Itemadapter.itemCount) - 1)
+            recyclerview.scrollToPosition((BookRecylerViewAdapter.itemCount) - 1)
```
Comment (Amar-2003) [inline app/src/main/java/com/example/insights/ForumnActivity.kt:33]: Revert Changes to this file

File: app/src/main/java/com/example/insights/HomeFragment.kt [hunk 2]
```diff
@@ -29,8 +29,8 @@ class HomeFragment : Fragment() {
                     for (change in value.documentChanges) {
                         if (change.type == DocumentChange.Type.ADDED) {
                             videoValues.add(change.document.data as HashMap<String, String>)
-                            val Itemadapter = ItemAdapter2(this, videoValues)
-                            recyclerView2.adapter = Itemadapter
+                            val BookRecylerViewAdapter = BookRecylerViewAdapter2(this, videoValues)
+                            recyclerView2.adapter = BookRecylerViewAdapter
                         }
```
Comment (Amar-2003) [inline app/src/main/java/com/example/insights/HomeFragment.kt:34]: Revert changes to this file

File: app/src/main/java/com/example/insights/ItemAdapter2.kt [hunk 3]
```diff
@@ -13,13 +13,13 @@ import androidx.cardview.widget.CardView
...
-    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ItemAdapter2.ViewHolder {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): BookRecylerViewAdapter2.ViewHolder {
         return ViewHolder(LayoutInflater.from(parent.context).inflate(R.layout.item_custom_row_home_video,parent,false))
     }
 
-    override fun onBindViewHolder(holder: ItemAdapter2.ViewHolder, position: Int) {
+    override fun onBindViewHolder(holder: BookRecylerViewAdapter2.ViewHolder, position: Int) {
         val item=items.get(position)
         holder.videoName.text=item["Title"]
         holder.itemView.setOnClickListener(object : View.OnClickListener {
```
Comment (Amar-2003) [inline app/src/main/java/com/example/insights/ItemAdapter2.kt:25]: Revert changes to this file

**Review by Amar-2003 — COMMENTED** [review]
Revert the other 3 files

**Comment by misaka3** [comment]
@Amar-2003 
Thank you for your message.
Reverted change of 3 files you said. PR number is #49.
Please confirm, thanks.

**Pull request closed without merging**